assets/scss/dropdown.scss

```diff
@@ -31,10 +31,12 @@
     text-decoration: none;
   }
 
-  &.more-facets {    
-    strong {
+  &.more-facets {
+    legend {
       font-size: 1.5rem;
       font-weight: 300;
```

Comment:
The font-weight can be removed as a legend has a font-weight of normal/300 by default.